google_photos_takeout_helper/__main__.py

```diff
@@ -216,7 +216,11 @@ def find_json_for_file(dir, file):
     # Returns date in 2019:01:01 23:59:59 format
     def get_date_from_folder_name(dir):
         dir = _os.path.basename(_os.path.normpath(dir))
-        dir = dir[:10].replace('-', ':') + ' 12:00:00'
+        dir = dir[:10].replace('-', ':').replace(' ', ':') + ' 12:00:00'
+
+        # Sometimes google exports folders without the -, like 2009 08 30...
+        # So the end result would be 2009 08 30 12:00:00, which does not match the format.
+        # Therefore, we also replace the spaces with ':'
```

Comment:
I'm afraid if some of those aren't "album folders" - but if you say that you have couple of hundreds of those, it's probably Google being Google

Wish that they don't do stuff like "2019  1  1" insted of "01", like it happens in exif :confounded: